pkg/storage/unified/resource/server.go

```diff
@@ -382,7 +406,7 @@ func (s *server) Create(ctx context.Context, req *CreateRequest) (*CreateRespons
 	if found != nil && len(found.Value) > 0 {
 		rsp.Error = &ErrorResult{
 			Code:    http.StatusConflict,
-			Message: "key already exists",
+			Message: "key already exists", // TODO?? soft delete replace?
```

Comment:
Why soft delete ?

Reply:
requires testing!  but I think, if you:

1. Create something with name `abc`
2. Delete `abc`
3. Create `abc`

you would get a "key already exists" error -- but I could be wrong.